Mathlib/AlgebraicGeometry/EllipticCurve/Affine.lean

```diff
@@ -821,16 +864,16 @@ lemma map_polynomialY : (W.map f).toAffine.polynomialY = W.polynomialY.map (mapR
   simp only [polynomialY]
   map_simp
 
-lemma map_eval_polynomialY (x : R[X]) (y : R) :
+lemma map_polynomialY_eval_eval (x : R[X]) (y : R) :
     ((W.map f).toAffine.polynomialY.eval <| x.map f).eval (f y) =
       f ((W.polynomialY.eval x).eval y) := by
```

Comment:
Shouldn't this statement also use `evalEval`?

Reply:
The pattern of evalEval is `(W.polynomialY.eval (C y)).eval x` so this statement doesn't exactly match the pattern. (We do occasionally evaluate at something that is not `C y`, for example `linePolynomial`.) An `evalEval` version could be deduced from this, but I'd add it when it's actually needed.

Note: I'm not adding deprecated aliases because this was only added recently in #13356.

Reply:
Actually, in the `universal` branch I'm now removing these lemmas in favor of bivariate polynomial lemmas
```
lemma map_mapRingHom_eval_map : (p.map <| mapRingHom f).eval (q.map f) = (p.eval q).map f

lemma map_mapRingHom_eval_map_eval (r : R) :
    ((p.map <| mapRingHom f).eval <| q.map f).eval (f r) = f ((p.eval q).eval r)

lemma map_mapRingHom_evalEval (x y : R) :
    (p.map <| mapRingHom f).evalEval (f x) (f y) = f (p.evalEval x y)
```